client/driver/utils.go

```diff
@@ -204,3 +205,17 @@ func SetEnvvars(envBuilder *env.Builder, fsi cstructs.FSIsolation, taskDir *allo
 		envBuilder.SetHostEnvvars(filter)
 	}
 }
+
+// getTaskKillSignal looks up the signal specified for the task if it has been
+// specified. If it is not supported on the platform, returns an error.
+func getTaskKillSignal(signal string) (os.Signal, error) {
+	if signal == "" {
+		return nil, nil
```

Comment:
Why not have it default to SIGINT

Reply:
I was debating whether it should be here or in executor.go- having the default here would be good; changing.